app/mapper/UserMapper.py

```diff
@@ -9,9 +8,8 @@ def __init__():
     pass
 
 
-def makeNew(name, password):
-    user = User(name, password)
```

Comment:
Gotta wonder how this was even working previously when they were passing the wrong number of params.